gulpfile.js

```diff
@@ -58,10 +59,15 @@ function getPlatforms(includeChromeOs) {
 
             break;
         case 'win32':
-            platforms.push('win32');
+			if (os.arch() == 'x64') {
```

Comment:
Better to add a separate `case 'win64':` instead of using an `if` statement to split codepaths for `win32` and `win64`.
After all you added `win64` to the possible values in `supportedPlatforms`.

Reply:
No, I can't. The code only enters here when the `supportedPlatforms` is empty (no platforms passed as parameters). The `switch` is against the `os.platform()` trying to detect the "actual" platform.